pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -95,6 +95,7 @@ type HintProvider interface {
 //Store interface is to allow Hint Providers to retrieve pod affinity
 type Store interface {
 	GetAffinity(podUID string, containerName string) TopologyHint
+	GetPolicy() Policy
```

Comment:
This seems like a strange addition to an interface called `Store`. I never actually liked the name `Store` to begin with but it was part of the original design before I got involved. I feel fairly strongly that this new function needs to live elsewhere.

Reply:
This is the interface which is passed to CPUManager from where i can deduce anything about Topology manager allowing me to fetch topology manager policy and perform validation checks in policy options for compatibility.

Reply:
OK. I took a closer look at this.

I think the issue is that we should not have an interface called `Store` in the topologymanager, and instead should call this interface something more generic (I don't have a good suggestion off the top of my head though). Let's leave it in here for now as you have done, and do a follow-up PR to rename this interface to something more appropriate.